modules/social_features/social_core/social_core.module

```diff
@@ -707,6 +707,10 @@ function social_core_field_widget_form_alter(&$element, FormStateInterface $form
 
     $element['#element_validate'] = ['social_core_path_validate'];
   }
+  if ($field_definition->getType() === 'address') {
+    $default_country = \Drupal::config('system.date')->get('country.default');
```

Comment:
What if the default country is not selected?

Reply:
Then the default value won't be set, meaning it will work as before.